jms/JMSSender/src/main/java/jakartaee/examples/jmssender/ScheduleService.java

```diff
@@ -0,0 +1,51 @@
+package jakartaee.examples.jmssender;
+
+import jakarta.annotation.PostConstruct;
+import jakarta.ejb.Schedule;
+import jakarta.ejb.Stateless;
+import jakarta.inject.Inject;
+import jakarta.jms.JMSException;
+import jakarta.json.Json;
+import jakarta.json.JsonObject;
+import jakarta.json.JsonWriter;
+import java.io.StringWriter;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.logging.Logger;
+
+/**
+ *
+ * @author Francisco Castillo (cefrancastillo@gmail.com)
+ */
+@Stateless
+public class ScheduleService {
+
+    private static Logger logger = Logger.getLogger(ScheduleService.class.getName());
+    
+    @Inject
+    private Sender sender;
+
+    @Schedule(second = "*/5", minute = "*", hour = "*")
+    public void sendMessage() throws JMSException {
+        try {
+            JsonObject jsonObject = Json.createObjectBuilder()
+                    .add("id", ThreadLocalRandom.current().nextInt(100))
+                    .add("status", ThreadLocalRandom.current().nextBoolean())
+                    .build();
+
+            StringWriter stWriter = new StringWriter();
+
+            JsonWriter jsonWriter = Json.createWriter(stWriter);
+            jsonWriter.writeObject(jsonObject);
+            jsonWriter.close();
+
+            String jsonData = stWriter.toString();
+
+            sender.sendMessage(jsonData);
+
+            logger.info("Mensaje enviado a cola.");
```

Comment:
Hi @Francisco-Castillo 
Wondering if this can be updated to English. I was looking around the code and all the other examples uses English as the main language.  For further improvements, I think we can open a Github issue to add internationalization to the project, wdyt?